GeoPol.xml

```diff
@@ -18,6 +18,7 @@
     <!-- Use back slash \ to indicate folder path e.g. src\external\ -->
     <ExcludeFolder>.gitignore</ExcludeFolder>
     <ExcludeFolder>GeoPol.xml</ExcludeFolder>
+    <ExcludeFolder>data</ExcludeFolder>
```

Comment:
miss a back slash "\"

Reply:
It has such an example `<!-- Use back slash \ to indicate folder path e.g. src\Test -->`. This should have the same effect, but I can add the back slash.